robolectric/src/test/java/org/robolectric/shadows/ShadowPowerManagerTest.java

```diff
@@ -115,4 +115,11 @@ public void isInteractive_shouldGetAndSet() {
     shadowPowerManager.setIsInteractive(false);
     assertThat(powerManager.isInteractive()).isFalse();
   }
+
+  @Test @Config(emulateSdk = Build.VERSION_CODES.LOLLIPOP)
```

Comment:
Robolectric always runs the tests with the latest API version that's supported. You shouldn't need to specify `emulateSdk` unless you're targeting an older API level.